e-commerce-project/src/app/components/login-page/login-page.component.html

```diff
@@ -1,8 +1,18 @@
 <h3>Welcom to login page!</h3>
 <p>Enter your username and password. Then push Submit button.</p>
 
-<form (ngSubmit)="toSubmit(username.value, password.value)">
-	<input #username type="text" name= 'username' placeholder="Username"><br><br>
-	<input #password type="password" name="pass" placeholder="Password"><br><br>
-	<button type="submit">Submit</button>
-</form>
\ No newline at end of file
+<form [formGroup]="loginForm" (ngSubmit)="toSubmit()">
+
+	<div class="">
+	  <label for="userName">Username:</label>
+	  <input type="email" formControlName="userName" class="" id="userName" placeholder="Username">
```

Comment:
1) `formControlName="userName"` 
The `formControlName` defines the name of the property of the input field (for reference) of `FormGroup` defined inside component (the property of form `formGroup` was mentioned above, which is defied inside component).

2) `type="email"` for the pattern (HTML5 type, not sure if it does anything in angular, but we just define it here for now, which means we are expecting login name to be in the pattern of email)